src/styles/base/_override.links.scss

```diff
@@ -1,49 +1,21 @@
-%link-inverse {
```

Comment:
Removed duplicative code from the links override while i was in there for focus styles anyways